src/test/java/CoinLocker/CoinLockerSystemTest.java

```diff
@@ -32,33 +39,53 @@ void test_addLockers() {
     }
 
     @Test
-    void test_rentLocker() {
+    void test_getAvailableLocker() {
         try {
             Locker locker = coinLockerSystem.getAvailableLocker(LockerSize.SMALL);
-            Assertions.assertNotEquals(null, locker);
+            Assertions.assertNotNull(locker);
             Assertions.assertEquals("CoinLocker.SmallLocker", locker.getClass().getName());
 
-            locker = coinLockerSystem.getAvailableLocker(LockerSize.SMALL);
-            Assertions.assertNotEquals(null, locker);
+            locker = coinLockerSystem.getAvailableLocker(LockerSize.MEDIUM);
+            Assertions.assertNotNull(locker);
             Assertions.assertEquals("CoinLocker.MediumLocker", locker.getClass().getName());
 
-            locker = coinLockerSystem.getAvailableLocker(LockerSize.SMALL);
-            Assertions.assertNotEquals(null, locker);
+            locker = coinLockerSystem.getAvailableLocker(LockerSize.LARGE);
+            Assertions.assertNotNull(locker);
             Assertions.assertEquals("CoinLocker.LargeLocker", locker.getClass().getName());
-
-            Assertions.assertThrows(NoAvailableLockerException.class, () -> coinLockerSystem.getAvailableLocker(LockerSize.SMALL));
+        } catch (InvalidLockerSizeException e) {
+            e.printStackTrace();
```

Comment:
Are we expecting these exceptions will be thrown this test? If so we should use assertThrows, if not, we could just add exceptions to the method declaration and let JUnit handle those exceptions.

Reply:
Changed